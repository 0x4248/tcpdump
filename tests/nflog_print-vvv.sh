#!/bin/sh

# NFLOG support depends on both DLT_NFLOG and working <pcap/nflog.h>

if grep '^#define HAVE_PCAP_NFLOG_H 1$' ../config.h >/dev/null
then
  ./TESTonce nflog_print-vvv nflog_print.pcap nflog_print-vvv.out '-t -vvv'
else
	printf '    %-30s: TEST SKIPPED (compiled w/o NFLOG)\n' 'nflog_print'
fi
