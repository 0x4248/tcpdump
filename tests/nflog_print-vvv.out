HOOK:OUT Prefix:out-icmp oif:10 MARK:0x2 UID:0 GID:100 IP (tos 0x0, ttl 64, id 24087, offset 0, flags [DF], proto ICMP (1), length 84)
    10.52.210.96 > 10.52.210.1: ICMP echo request, id 2995, seq 1, length 64
HOOK:IN Prefix:in-icmp iif:10 MARK:0x1 HWHDR=e89a8fa8a7bc:6805ca0251b2:0800 IP (tos 0x0, ttl 64, id 39507, offset 0, flags [none], proto ICMP (1), length 84)
    10.52.210.1 > 10.52.210.96: ICMP echo reply, id 2995, seq 1, length 64
HOOK:OUT Prefix:out-dns oif:6 MARK:0x4 UID:1000 GID:100 IP (tos 0x0, ttl 64, id 9134, offset 0, flags [none], proto UDP (17), length 51)
    10.0.0.64.49997 > 10.0.0.1.53: [udp sum ok] 2830+ A? ya.ru. (23)
HOOK:IN Prefix:in-dns iif:6 MARK:0x3 HWHDR=e89a8fa8a7bc:6805ca0251b3:0800 IP (tos 0x0, ttl 64, id 8040, offset 0, flags [none], proto UDP (17), length 250)
    10.0.0.1.53 > 10.0.0.64.49997: [udp sum ok] 2830 q: A? ya.ru. 3/2/4 YA.ru. [1m54s] A 93.158.134.3, YA.ru. [1m54s] A 213.180.204.3, YA.ru. [1m54s] A 213.180.193.3 ns: YA.ru. [1h4m20s] NS ns1.yandex.RU., YA.ru. [1h4m20s] NS ns2.yandex.RU. ar: ns1.YANDEX.ru. [15h47m28s] A 213.180.193.1, ns2.YANDEX.ru. [4h34m25s] A 93.158.134.1, ns1.YANDEX.ru. [1d23h35m14s] AAAA 2a02:6b8::1, ns2.YANDEX.ru. [1d23h35m14s] AAAA 2a02:6b8:0:1::1 (222)
